editor/src/clj/editor/defold_project.clj

```diff
@@ -682,6 +682,10 @@
   (cond
     (.equalsIgnoreCase "nearest" s) gl/nearest
     (.equalsIgnoreCase "linear" s) gl/linear
+    (.equalsIgnoreCase "nearest_mipmap_nearest" s) gl/nearest-mipmap-nearest
+    (.equalsIgnoreCase "nearest_mipmap_linear" s) gl/nearest-mipmap-linear
+    (.equalsIgnoreCase "linear_mipmap_nearest" s) gl/linear-mipmap-nearest
+    (.equalsIgnoreCase "linear_mipmap_linear" s) gl/linear-mipmap-linear
```

Comment:
I missed that the same logic should be added for the editor. I added new filter types in filter parser, but now Editor should behave the same as runtime:
if mipmaps turned off for the atlas, non-mipmap filter should be picked autimatically.